index.html

```diff
@@ -8,6 +8,55 @@
     <title>Smart Agency</title>
 </head>
 <body>
+    <header class="background-activ">
+        <div class="g-container">
+            <div class="header-logo-menu">
+                <div class="logo">
+                    <p><span>S</span>mart</p>
+                </div>
+                <div id = "headerMenu" class="header-menu">
+                    <ul>
+                        <li><a href = "index.html">HOME</a></li>
+                        <li><a href = "index.html">SERVIES</a></li>
+                        <li><a href = "index.html">PORTFOLIO</a></li>
+                        <li><a href = "index.html">ABOUT</a></li>
+                        <li><a href = "index.html">CONTACT</a></li>
+                    </ul> 
+                </div>
+                <div class="contain-burger">
+                    <div class="burger" id = "burgerIcon">
+                        <div class="burger-icon icon1"></div>
```

Comment:
don't use names like "icon1", "icon2", etc..
If it's just line, you can name it as "burger-line", or "menu-item"